pkg/controller/stacks/install/stackinstall.go

```diff
@@ -373,7 +373,7 @@ func (h *stackInstallHandler) delete(ctx context.Context) (reconcile.Result, err
 
 	// Delete all StackDefintions created by this StackInstall or
 	// ClusterStackInstall
-	if err := h.kube.DeleteAllOf(ctx, &v1alpha1.StackDefinition{}, client.InNamespace(h.ext.GetNamespace()), client.MatchingLabels(labels)); runtimeresource.IgnoreNotFound(err) != nil {
+	if err := h.kube.DeleteAllOf(ctx, &v1alpha1.StackDefinition{}, client.InNamespace(h.ext.GetNamespace()), client.MatchingLabels(labels)); err != nil {
```

Comment:
The `DeleteAllOf` cleanup is in a separate commit and can be removed if it creates too much noise.

This is a response to https://github.com/crossplane/crossplane/pull/1258#discussion_r379293273 and 
https://github.com/crossplane/crossplane/pull/1217#discussion_r376425771